src/NewCommand.php

```diff
@@ -38,8 +38,23 @@ class NewCommand extends Command
      * The boilerplate namespace of the plugin
      * @var string
      **/
+    
+    const PLUGIN_SLUG = 'plugin-name';
+
     const PLUGIN_NAMESPACE = 'Vendor\PluginName';
 
+    const DEF_PLUGIN_DESCRIPTION = 'Description: A description of this plugin';
```

Comment:
I assume DEF stands for default?

That change makes sense, since it's more expressive. However, I'd prefer to use DEFAULT instead of DEF. As a general rule I steer clear of abbreviations as they tend to cause more harm than good in my experience.